gensim/models/nmf.py

```diff
@@ -588,11 +588,18 @@ def update(self, corpus, chunksize=None, passes=None, eval_every=None):
         if isinstance(corpus, collections.Iterator) and self.passes > 1:
             raise ValueError("Corpus is an iterator, only `passes=1` is valid.")
 
-        logger.info(
-            "running NMF training, %s topics, %i passes over the supplied corpus of %s documents, evaluating l2 norm "
-            "every %i documents",
-            self.num_topics, passes, lencorpus, evalafter,
-        )
+        if np.isinf(lencorpus):
```

Comment:
I don't think this is better than what is already in develop. You're adding a conditional, fingering the type of a variable, and duplicating 5 lines to achieve a slightly different logging message. The same thing applies to the logging statement below.

Regardless whether lencorpus is an integer or np.inf, then there is no harm in logging it as a str.

Am I missing something here?

Reply:
Well, I thought that `...passes over the supplied corpus of inf documents...` wouldn't be quite clear to a user, but yeah, I suppose we can leave it as is.

Reply:
@anotherbugmaster Weren't we going to revert this?